user/urls.py

```diff
@@ -1 +1,11 @@
-# write your code here
+from django.urls import path
+
+from user.views import CreateTokenView, CreateUserView, ManageUserView
+
+urlpatterns = [
+    path("register/", CreateUserView.as_view(), name="create"),
+    path("login/", CreateTokenView.as_view(), name="login"),
+    path("me/", ManageUserView.as_view(), name="manage"),
+]
+
+app_name = "user"
```

Comment:
Make sure you've added a blank line at the end of the file as per the code style checklist.